app/src/main/java/com/yeonae/chamelezone/view/login/ChangePasswordFragment.kt

```diff
@@ -0,0 +1,112 @@
+package com.yeonae.chamelezone.view.login
+
+import android.os.Bundle
+import android.os.Handler
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.yeonae.chamelezone.App
+import com.yeonae.chamelezone.Injection
+import com.yeonae.chamelezone.R
+import com.yeonae.chamelezone.ext.shortToast
+import com.yeonae.chamelezone.view.login.presenter.ChangePasswordContract
+import com.yeonae.chamelezone.view.login.presenter.ChangePasswordPresenter
+import kotlinx.android.synthetic.main.fragment_change_password.*
+import kotlinx.android.synthetic.main.fragment_find_password.btn_back
+import java.util.regex.Pattern
+
+class ChangePasswordFragment : Fragment(), ChangePasswordContract.View {
+    override lateinit var presenter: ChangePasswordContract.Presenter
+    var memberNumber = 0
+    var lastInputTime = 0L
+
+    override fun showResultView(response: Boolean) {
+        if (response) {
+            context?.shortToast(R.string.changed_pw)
+        }
+        (activity as? LoginActivity)?.back()
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return inflater.inflate(R.layout.fragment_change_password, container, false)
+    }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        memberNumber = arguments?.getInt(MEMBER_NUMBER) ?: 0
+
+        presenter = ChangePasswordPresenter(
+            Injection.memberRepository(App.instance.context()), this
```

Comment:
```suggestion
            Injection.memberRepository(), this
```
주입하지 말고 Injection 내부에서 App.instance.context() 를 사용하는 것이 좋을 것 같아요.
타입이 Context이므로 다른사람이 사용할 시에 applicationContext가 아닌 context를 주입하는 경우가 있을 수 있고,
이런 부분은 지난 리뷰에서 이미 발생했었습니다.

Reply:
수정했습니당